src/redux/wallets.ts

```diff
@@ -233,7 +230,9 @@ export const walletsLoadState = (profilesEnabled = false) => async (
     profilesEnabled && dispatch(fetchWalletENSAvatars());
     return wallets;
   } catch (error) {
-    legacyLogger.sentry('Exception during walletsLoadState');
+    logger.error(new RainbowError('Exception during walletsLoadState'), {
+      message: (error as Error)?.message,
+    });
     captureException(error);
```

Comment:
Can remove `captureException` uses too since `logger.error` does this too.

Reply:
Fixed!